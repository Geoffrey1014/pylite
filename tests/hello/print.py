print 123
print
print "STRING"
print None
print True
print False
