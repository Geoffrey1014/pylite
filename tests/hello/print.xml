lambda
  name __init__
  param_count 0
  local_count 0
  label_count 0
  insn_vector
    load_const c0
    print
    load_const c1
    print
    load_const c2
    print
    load_const c3
    print
    load_const c4
    print
    load_const c5
    print
    load_const c3
    return
  const_vector
    123.000000
    ""
    "STRING"
    None
    True
    False
  guard_vector
  line_number_map 1 1 0 0 3 3 4 4 5 5 6 6 1 1
