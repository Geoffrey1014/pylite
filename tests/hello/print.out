123

STRING
None
True
False
